src/Foreman.php

```diff
@@ -27,6 +27,7 @@ class Foreman implements ForemanInterface
     use Update\Complete\Success\Factory\AwareTrait;
     use Defensive\AwareTrait;
     use Logger\AwareTrait;
+    use Logger\Message\Metadata\Builder\AwareTrait;
```

Comment:
This will inject the prototype builder that the factory clones, modifying that prototype is kind of dangerous. I'd rather we made the builder a `shared: true` actor and did away with the `Factory`